src/ngraph/runtime/gpu/gpu_emitter.cpp

```diff
@@ -335,17 +336,28 @@ void runtime::gpu::GPU_Emitter::emit_BatchNorm(EMIT_ARGS)
                                                 args[2].get_shape(),
                                                 args[0].get_shape(),
                                                 batchnorm->get_eps_value(),
-                                                global_stats);
+                                                global_stats,
+                                                save_stats);
 
     writer.block_begin();
     {
-        writer << "void* input[] = {" << node_names(args) << "};\n";
-        writer << "void* output[] = {" << node_names(out) << "};\n";
+        writer << "void* input[] = {" << runtime::gpu::GPU_Emitter::node_names(args) << "};\n";
+        writer << "void* output[] = {" << runtime::gpu::GPU_Emitter::node_names(out) << "};\n";
         writer << "gpu::invoke_primitive(ctx, " << index << ", input, output);\n";
     }
     writer.block_end();
 }
 
+void runtime::gpu::GPU_Emitter::emit_BatchNorm(EMIT_ARGS)
+{
+    ::emit_BatchNorm(external_function, writer, node, args, out, false);
+}
+
+void runtime::gpu::GPU_Emitter::emit_CUDNNBatchNorm(EMIT_ARGS)
+{
+    ::emit_BatchNorm(external_function, writer, node, args, out, true);
```

Comment:
name it something like 'emit_BatchNormWithSavedMeanVar'?

Reply:
How about `emit_CUDNNBatchNorm -> emit_BatchNormWithStats`. We change the op name too? `CUDNNBatchNorm` -> `BatchNormWithStats`? 

Note that when the new API for batch norm goes in, it would be `BatchNormTraining` and `BatchNormTrainingWithStats`

Reply:
Good.